src/core/collision_component.cpp

```diff
@@ -59,9 +71,21 @@ void CollisionComponentLoader::BindValues()
         Bind<CollisionClass::Type>( &CollisionComponent::SetCollisionClass, collisionClass( AutoId( istr ) ) );
     }
     Bind("clip_scene", func_bool(&CollisionComponent::SetClipScene));
+    Bind("dynamic_radius", func_bool(&CollisionComponent::SetDynamicRadius));
 
 }
 
+void CollisionComponentLoader::FillProperties( ComponentHolder& actor ) const
+{
+    bool dynradius = false;
+    if( mSetters != nullptr )
+    {
+        Json::GetBool( (*mSetters)["dynamic_radius"], dynradius );
+    }
+    ComponentLoader<CollisionComponent>::FillProperties( actor );
+    actor.Get<ICollisionComponent>()->SetDynamicRadius( dynradius );
```

Comment:
yep, tried it. if there's no entry in the json for dyn radius, the setter is not called.